README.md

```diff
@@ -26,28 +27,28 @@ To build for any of the above, run:
 ./go.sh <arch>
 ```
 
-Production builds are done using the script in `.shippable.yml` which
-identifies some packages you'll need to have installed for the above
-command to work.
+Production builds are done using the scripts in `.buildkite/` and
+`.github/workflows/` which identify some packages you'll need to
+have installed for the above command to work.
 
 When building locally be aware that `./go.sh` is written assuming it's
 invoked in a pristine workspace.  That means if you pull an update of
-this repository `./go.sh` will not update the `crosstool-ng` source and
-rebuild the `bin/ct-ng` utility to match the update configuration.  This
+this repository `./go.sh` will not update the `crosstool-ng` source or
+rebuild the `bin/ct-ng` utility to match the updated configuration.  This
 can produce very confusing results.
 
 ## Released Binaries packages
 
-Pre-built binaries releases of the SDK are made and currently only available for
-an x86_64 Linux host environment.  The releases can be found here:
+Pre-built binary releases of the SDK are made for a few OSes and host
+architectures. They can be found here:
 
 https://github.com/zephyrproject-rtos/sdk-ng/releases
 
 ## PR Builds
 
 Successfull PR builds will get posted here:
 
-https://builds.zephyrproject.org/zephyrproject-rtos/sdk-ng/<PR NUMBER>/zephyr-sdk-<VERISON>-setup.run
+https://builds.zephyrproject.org/zephyrproject-rtos/sdk-ng/&lt;PR NUMBER>/zephyr-sdk-&lt;VERISON>-setup.run
```

Comment:
It looks like `&lt;` is not being properly rendered in the GitHub flavoured Markdown; though, it certainly is an improvement over what we had earlier.
https://github.com/zephyrproject-rtos/sdk-ng/blob/de50dadd3c2645e18c1a2722d2fb565f0b97353a/README.md

Noting that this text is not meant to be an actual link, we can just treat this as "code":
```suggestion
`https://builds.zephyrproject.org/zephyrproject-rtos/sdk-ng/<PR NUMBER>/zephyr-sdk-<VERISON>-setup.run`
```

Reply:
> It looks like &lt; is not being properly rendered in the GitHub flavoured Markdown

Yes, seems to be not recognized as part of autodetected URLs (second `&lt;`, not recognized as continuation of a link, was rendered properly).

> Noting that this text is not meant to be an actual link, we can just treat this as "code":

Did it like that.